src/javascript/app_2/pages/settings/sections/cashier_password.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Section from '../components/section.jsx';
+
+class CashierPassword extends PureComponent {
```

Comment:
Maybe now can make all the components & sections to functional components. 

`export const CashierPassword = ({title, description}) => ();`

But later! 👍